docs/reset-security-verification-for-wog-account.md

```diff
@@ -1,23 +1,23 @@
 # Reset MFA for WOG account
 
-This article guides you how to reset your WOG(SGGovt M365) MFA.
+This article provides instructions on how users can reset their WOG MFA(Multi-Factor Authentication). The WOG profile is listed as SGGovt M365 on the authenticator app. 
 
 ## Audience
 
-Users who have a WOG profile(account) may want to reset the MFA of their WOG account for any of the following reasons:
+Users who have a WOG profile, listed as SGGovt M365 on the authenticator app, may need to reset the MFA for the following reasons:
 
 - Changed mobile phone.
 - Lost mobile phone.
 - Deleted the Authenticator app from the mobile phone or unable to access the app on your phone.
-- Change in the organisational email address linked to the WOG account.
+- Change in the organisational email address linked to the WOG profile.
 
-?> You may also need to [reset MFA of your TechPass account](reset-security-verification-for-wog-account).
+?> You also need to [reset MFA of your TechPass account](reset-security-verification-for-wog-account).
 
 ## Prerequisites
 
 - You must have the old mobile phone to authenticate your WOG login.
 
-?> If you have lost your old mobile phone, formatted the old phone(factory reset), or unable to authenticate your login using the old mobile phone, contact your Agency Facility Management(AFM) to remove the MFA configured for your WOG account. They will get provide the steps to reset your WOG MFA and also your TechPass MFA.
```

Comment:
"old mobile phone", no idea, is there a better way to call it?

Reply:
I have changed this to "?> If you have lost or formatted your previous mobile phone or are unable to authenticate using it, contact your Agency Facility Management(AFM) to remove the MFA configured for your WOG profile."

Pls let me know , if this works. Thanks!